include/seqan3/search/algorithm/detail/search_trivial.hpp

```diff
@@ -84,19 +84,15 @@ namespace seqan3::detail
 {
 
 template <bool abort_on_hit>
-auto _search_trivial(auto it, auto const & query, uint64_t const query_pos, search_params error_left,
-                     auto && delegate)
+bool _search_trivial(auto it, auto & query, uint64_t const query_pos, search_params error_left, auto && delegate)
```

Comment:
Use of `auto` typed parameters is valid as of the new concepts syntax, too, but the use in this file is very inconsistent (sometime `typename query_type`, sometime `auto`).

Reply:
Why was `const` removed on the query?